CSA.md

```diff
@@ -1,215 +1,221 @@
 # Cloud Service Agreement
 
 ## USING THIS AGREEMENT
-This Agreement allows <span class="coverpage_link">Customer</span> to access and use the Cloud Service. To use this Agreement, the parties must complete and sign or electronically accept a Cover Page and Order Form.
-Capitalized and highlighted words have the meanings given on the Cover Page or in the Order Form. However, if the Cover Page or Order Form omit or do not define a highlighted word, the default meaning will be "none" or "not applicable". All other capitalized words have the meanings given in these Standard Terms.
+To use this Agreement, the parties must complete and sign or electronically accept a Cover Page.
+Capitalized and <span class="keyterms_link">highlighted</span> words have the meanings given on the Cover Page. However, if the Cover Page omits or do not define a highlighted word, the default meaning will be "none" or "not applicable". All other capitalized words have the meanings given in these Standard Terms.
```

Comment:
The staged version shows the outdated text in the yellow box. Not sure how that happened, since this contains the proper text?

Reply:
It's because of how that box works in Wordpress. I think Lauren set it up outside of the content.